### PR TITLE
GW 2

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
I've had some weird issues with this PR. I messaged Jared about them, but I also noticed I have some lint errors, but I cannot see them in asciidoc and I'm not sure if there is any other way to find them?

File: journal-temp/modernized/02-chapter-2.adoc [hunk 1]
```diff
@@ -177,9 +184,14 @@ for my giving him the title of T. Waldgrave, in my letter to him,
 according to his own subscription to his warrant of commitment, as is hereafter shown.
 A poor pretence for detaining me fifteen months in prison!
 
-The substance of a letter which I sent to Thomas Waldgrave,
-after he committed me to Bury jail.
+[.offset]
+_The substance of a letter which I sent to Thomas Waldgrave,
```
Comment (maijapapaya) [inline journal-temp/modernized/02-chapter-2.adoc:188]: I wasn't sure how to handle these things here, and there are several of them. They are italicized in the ODT. I tried different things and this is what I settled on. Maybe there is some cut and dry way to do them that I have forgotten?? 😄

Comment (maijapapaya) [reply journal-temp/modernized/02-chapter-2.adoc:188]: So, scratch that, I think I found the lint error while looking at the PR. If there are more, lemme know!

**Comment by maijapapaya** [comment]
So, scratch that message about the lint error, I think I found it while looking at the PR. If there are more, lemme know!

**Comment by Henderjay** [comment]
Hey Maija, there was one place where you forgot to put the ==== to close the numbered section. I fixed that for you and merged. The other decisions you made about those unusual letter titles or intros I thought looked good 👍🏻.

**Pull request merged**